pkg/proto/nbcontract/v1/customcatrustconfig.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
+package nbcontract.v1;
+import "pkg/proto/nbcontract/v1/featurestate.proto";
+
+message CustomCaTrustConfig {
+    FeatureState status = 1;
+    repeated string custom_ca_certs = 2;
+  }
```

Comment:
Please check on best practices for casing abbreviations in this context (should it be CustomCATrustConfig?), and using plurals for repeated fields (should it be "cert" or "certs"). The current change may be right, just to double check.

Reply:
Here are what I tried.
If we put _CustomCaTrustConfig custom_ca_trust_config = 33;_ in config.proto, 
in the compiled .go, it will be _CustomCaTrustConfig           *CustomCaTrustConfig_
The type and name are consistent.

I don't have a strong preference on this. We can still do _CustomCATrustConfig_.
p.s., _**Field definitions in the .proto files must use lower_case_underscore_separated_names**_ is suggested by Google.

Thoughts?

Reply:
Btw, plural form of repeated field is suggested, ie. certs.

Reply:
> I don't have a strong preference on this. We can still do CustomCATrustConfig.

Thats what Golang style for [initialisms](https://google.github.io/styleguide/go/decisions.html#initialisms) in identifiers prescribes. (This is often checked by linters too ...)